src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryText.cs

```diff
@@ -255,6 +257,66 @@ private static string ReplaceWith24HourClock(string format)
             return Regex.Replace(format, pattern, "HH${nonHours}", RegexOptions.IgnorePatternWhitespace);
         }
 
+        /// <summary>
+        /// Resolves month/minute ambiguity and 24h vs AM/PM.
+        /// </summary>
+        /// <param name="format">Format string.</param>
+        /// <returns></returns>
+        private static string ResolveDateTimeFormatAmbiguities(string format)
+        {
+            if (format.Length == 0)
+            {
+                return format;
+            }
+
+            // Handling especial cases
```

Comment:
>Handling especial cases [](http://example.com/codeflow?start=15&length=23)

It is good to add that we are adding the '%' because that's what .NET requires (I didn't know that until I tried it today)

Reply:
Right. Since PA is case insensitive, a single `d` would result in a totally different format than a day number in .NET.
I'll make sure to add the comment.

Reply:
This have been removed and is not long part of this PR.